certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -2893,77 +2869,342 @@
+                  "title": ""
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "Added By MS Power Automate"
+                }
+              },
+              "required": [
+                "callBackUrl",
+                "secret",
+                "typeName",
+                "description",
+                "filters"
+              ]
+            }
+          }
+        ],
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/api/v1/Individuals/{id}/Customfields": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}
+          }
+        },
+        "summary": "Update Custom Field Value",
+        "description": "Update Custom Field Value",
+        "operationId": "Update-custom-field-value",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Customer ID",
+            "x-ms-summary": "Customer ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "Custom Field Name",
+                  "title": "Custom Field Name"
+                },
+                "value": {
+                  "type": "string",
+                  "description": "Custom Field Value. For multi-selects, use \"|\" as delimiter",
+                  "title": "Custom Field Value"
+                }
+              },
+              "required": [
+                "name",
+                "value"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "IndividualData": {
+      "type": "object",
+      "properties": {
+        "firstName": {
+          "type": "string",
+          "title": "First Name",
+          "description": "FirstName"
+        },
+        "lastName": {
+          "type": "string",
+          "title": "Last Name",
+          "description": "LastName"
+        },
+        "middleName": {
+          "type": "string",
+          "title": "Middle Name",
+          "description": "MiddleName"
+        },
+        "preferredFirstName": {
+          "type": "string",
+          "title": "Preferred First Name",
+          "description": "PreferredFirstName"
```

Comment:
@RjImpx The description values should be free from grammatical and spell mistakes. All similar the description values need to be fixed as they are visible to public.